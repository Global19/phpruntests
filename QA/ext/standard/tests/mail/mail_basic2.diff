003+ bool(false)
003- bool(true)