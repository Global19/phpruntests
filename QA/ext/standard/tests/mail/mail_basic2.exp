*** Testing mail() : basic functionality ***
-- extra parameters --
bool(true)
--- Extras
===DONE===