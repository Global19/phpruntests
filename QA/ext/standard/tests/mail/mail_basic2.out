*** Testing mail() : basic functionality ***
-- extra parameters --
bool(false)
--- Extras
===DONE===